src/EnergyPlus/ThermalComfort.cc

```diff
@@ -715,6 +718,16 @@ namespace ThermalComfort {
 			} else {
 				ShowSevereError( "PEOPLE=\"" + People( PeopleNum ).Name + "\", Incorrect Clothing Type" );
 			}}
+			// for unit testing
```

Comment:
Not a big fan of adding special code for unit tests.  Can't you add some Schedule:Constant(s) to the unit test setup?